web/src/Storage.jsx

```diff
@@ -71,9 +71,10 @@ export default function Storage() {
         candidateDevices: [disk]
       } = await client.storage.getStorageProposal();
       const actions = await client.storage.getStorageActions();
+      const target = disk || disks[0]?.id;
```

Comment:
NP: Not sure if worth it adding a reminder about _"`disks` is a collection of plain JavaScript objects while `disk` is just an id"_.

Reply:
Thinking it twice, maybe using _diskId_ instead _disk_ [while destructuring  `candidateDevices`](https://github.com/yast/d-installer/pull/180/files#diff-7e1258ac61a3996cefbc2a9726cd5962b3767c14e55817d7f5351cd97db2c2a5R71) could help more to future readers since that line will become


```suggestion
      const target = diskId || disks[0]?.id;
```